XU-TechnicalPaper.md

```diff
@@ -229,6 +233,22 @@ placeOrder(LTC/ETC, 0, 10)
 -------------
 The Lightning Network and the Raiden Network take care of security of funds on the payment channel rails. This includes resolving disputes over balances on the underlying blockchain and disincentivizing malicious behavior by POS-style punishment (a dishonest party will lose all her funds to the honest party). It especially also includes security when routing payments through intermediary hops. Atomic Swaps bind two payments of a trade together, so that in no event one party can disappear after receiving the first part of the trade. Both payments in a trade have to complete successfully, for both payments to be valid and spendable. Exchange Union is secured against malicious order information in a similar POS-style punishment manner, to disincentivize market manipulation by sending out false order information.
 
+XUD's ID Model
+=============================================================
+
+During setup, a XUD automatically creates a public/private key pair (using bitcoin's `secp256k1 ECDSA`) for identification and order signing purposes, the `XUD_ID`. The private key is saved on disk and encrypted with a password. A reboot of XUD requires entering this password; generating new public/private key pairs is optional. The public key represents one XUD's unique ID within Exchange Union. Exchanges can let others know what their XUD's public key is by making such available on their website. In this way, exchanges in jurisdictions which require licenses can create a white-list and only connect to other exchanges which fulfill the local jurisdiction's legal requirements. The same applies to implementing compliant KYC and AML policies, which is every exchange's responsibility. This is achieved by signing orders.
```

Comment:
I'd say either "XUD automatically creates" or "a node automatically creates." Also this key is not referred to as `XUD_ID` anywhere in the code, but I suppose we can start calling it that.

What do you mean by "generating new public/private key pairs is optional?" I'm thinking this key should be preserved and tied to the staking described above.

Reply:
Yeah changed that. At least for the PoC we don't cover this. If one forgets the password, one has to manually delete the key files from the filesystem to automatically generate new ones on reboot. And yes, staked XUC are gone then.